src/ngx_http_lua_timer.c

```diff
@@ -559,6 +559,13 @@ ngx_http_lua_timer_handler(ngx_event_t *ev)
     ngx_http_lua_main_conf_t        *lmcf;
     ngx_http_core_loc_conf_t        *clcf;
 
+    lua_Debug                ar;
+    u_char                  *p;
+    u_char                   errbuf[64];
+    int                      errbuf_size = 64;
```

Comment:
Better avoid repeating this magical number 64, which can easily get out of sync. Better use a local anonymous `enum {}` to give this constant a name?

This way we can also save the `errbuf_size` auto variable.

Also, I don't think `64` is large enough. Better be 128 or even larger? Just to be safe.